backend/requirements.txt

```diff
@@ -18,23 +18,23 @@ django-oauth-toolkit==1.7.1
 django-prometheus==2.2.0
 # Registration, profile, password and email management
 # https://github.com/apragacz/django-rest-registration
-django-rest-registration==0.7.3
-# Useful helpers for quesries such as SubqueryCount
+django-rest-registration==0.8.2
+# Useful helpers for queries such as SubqueryCount
 django-sql-utils==0.6.1
 # Django REST Framework is used for most of the API routes
 # https://github.com/encode/django-rest-framework
 djangorestframework==3.14.0
 # Used to generate OpenAPI documentation exposed by Swagger UI
 # See https://github.com/tfranzel/drf-spectacular
-drf-spectacular==0.26.0
+drf-spectacular==0.26.4
 # Pillow is used to generate the Previews for links shared on social media
-Pillow==9.4.0
+Pillow==9.5.0
 # Needed for postgres database
-psycopg2-binary==2.9.5
+psycopg2-binary==2.9.7
 # PyYAML is used for reading the settings 
 PyYAML==6.0
 # tqdm is used to display a progressbar in some slow ML algorithms
-tqdm==4.65.0
```

Comment:
Note that this dependency is used only in `Entity.recompute_quantiles()` which is obsolete, and could probably be removed completely.